payment-request/payment-request-show-method.https.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
-<!-- Copyright © 2017 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
 <meta charset="utf-8">
 <title>Test for PaymentRequest.show() method</title>
 <link rel="help" href="https://w3c.github.io/browser-payment-api/#show-method">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
```

Comment:
You'll need to pull in `/resources/testdriver.js` here; currently failing with

> `promise_test: Unhandled rejection with value: object "ReferenceError: test_driver is not defined"`

https://travis-ci.org/web-platform-tests/wpt/jobs/404559383

Reply:
And, `testdriver-vendor.js` (empty, but enforced by a lint rule)

Reply:
Ah, don't know how i missed that :/